### DISCUSSION TITLE
How to change colour of slash between num-filtered-rows and num-rows?

### DISCUSSION THREAD

**Discussion by nathanielevan** [davatorium/rofi#1586] [Theming] [answered]
As described on title. Also, what's the default grey colour used by the placeholder text, num-rows and num-filtered-rows, among possibly other things, and is there a way to change all of them at the same time?

**Comment by DaveDavenport** [comment, marked as answer]
Not sure what the actual question is. Everything you mention is part of the theme and can be changed using the theme system.

See the rofi-thme manpage: https://github.com/davatorium/rofi/blob/next/doc/rofi-theme.5.markdown

You did not specify version of rofi or what theme you use, but in the default theme:

separator: https://github.com/davatorium/rofi/blob/next/doc/default_theme.rasi#L138
color: https://github.com/davatorium/rofi/blob/next/doc/default_theme.rasi#L139
https://github.com/davatorium/rofi/blob/next/doc/default_theme.rasi#L134

how to override these specific elements, see manpage.

**Reply by nathanielevan** [reply]
Actually that's precisely what I'm looking for, thanks! Sorry for not being clear enough. I should mention I'm using the latest version (1.7.3) installed through the Arch community repository.
